### DISCUSSION TITLE
How do you reset a textarea with multiple returns/linebreaks?

### DISCUSSION THREAD

**Discussion by centinel3** [jaredpalmer/formik#3436] [General] [answered]
I have a textarea that I allow users to type into and create paragraphs with. When I rest the form using resetForm({}); it clears the values, but leaves an empty return/line break in the textarea. 

Here is my code for the form:
`
<Formik
        initialValues={{
          body: "",
        }}
        onSubmit={(
          values: CommentFormType,
          { resetForm, setSubmitting }: FormikHelpers<any> // TODO: fix the type here. FeedbackFormType throws a type error
        ) => {
          setSubmitting(false);

          // postReply({
          //   variables: {
          //     body: values.body,
          //   },
          // });

          resetForm({});
        }}
        validateOnChange
        validationSchema={yup.object().shape<CommentFormType>({
          body: yup.string(),
        })}
      >
`
The textarea has a placeholder set and when the form submits, it is not present due to the return/linebreak. As soon as you remove it, placeholder appears. 

Any thoughts would be appreciated. -Chris

**Comment by centinel3** [comment, marked as answer]
Figured it out. I wasn't preventDefault() on the field. works as expected after I added that.

**Reply by t-ashraf** [reply]
For clarity,

```
const handleOnKeyDown(evt: React.KeyboardEvent<HTMLTextAreaElement>) => {
    if (evt.code === 'Enter') {
      evt.preventDefault();
      // the rest of your code here based on 'Enter' key...
    }
  };
```
